fortnite_api/shop.py

```diff
@@ -142,6 +145,29 @@ def from_value(cls: Type[Self], value: str, /) -> Self:
         )
 
 
+class ShopEntryOfferTag(ReconstructAble[Dict[str, Any], HTTPClientT]):
```

Comment:
This class should also inherit `abc.Hashable` for `hash(x)` and comparison operations. Documentation should be updated accordingly too.